components/console-backend-service/Dockerfile

```diff
@@ -1,43 +1,39 @@
-FROM golang:1.11.4-alpine3.8 as builder
+FROM eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190930-d28d219 as builder
```

Comment:
Curiosity: why not `golang-alpine`?

Reply:
this image is quite old btw, you use go: 1.13 in go mod, here you have 1.12, and newest go version is 1.14
![image](https://user-images.githubusercontent.com/17271979/84026270-4b7a2380-a98d-11ea-8dc6-2542b6860c44.png)

![image](https://user-images.githubusercontent.com/17271979/84026252-44ebac00-a98d-11ea-9d8d-b1f758291207.png)
 is 1.14

Reply:
We use buildpack because it contains some tooling like goimports already vendored.

Reply:
And I have to revert to go 1.12 in go mod, that's true :)